client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -115,6 +115,7 @@ export const PatientResultsTab: FC<PatientPanel> = ({ patient, value }) => {
         columns={columns}
         noDataMessage={t('messages.no-matching-patients')}
         onRowClick={row => {
+          setNewPatient(undefined);
```

Comment:
Sneaked this in. If patient is found and selected the existing patient wasn't loaded correctly.